integrations/emscripten.rst

```diff
@@ -0,0 +1,150 @@
+.. _emscripten:
+
+|emscripten_logo| Emscripten
+____________________________
+
+It should be possible to build packages for `Emscripten <https://emscripten.org/>`__ (`asm.js <http://asmjs.org/>`_) via the following conan profile:
+
+.. code-block:: text
+
+  include(default)
+  [settings]
+  os=Emscripten
+  arch=asm.js
+  compiler=clang
+  compiler.version=6.0
+  compiler.libcxx=libc++
+  [options]
+  [build_requires]
+  emsdk_installer/1.38.29@bincrafters/stable
+  [env]
+
+And the following conan profile is required for the `WASM <https://webassembly.org/>`_ (Web Assembly):
+
+.. code-block:: text
+
+  include(default)
+  [settings]
+  os=Emscripten
+  arch=wasm
+  compiler=clang
+  compiler.version=6.0
+  compiler.libcxx=libc++
+  [options]
+  [build_requires]
+  emsdk_installer/1.38.29@bincrafters/stable
+  [env]
+
+These profile above are using the `emsdk_installer/1.38.29@bincrafters/stable <https://github.com/bincrafters/conan-emsdk_installer>`_ conan package.
```

Comment:
Not yet in conan-center, maybe it is worth to remember link how to add the _bincrafters_ remote

Reply:
added (also submitted an inclusion request)